universal_transfer_operator/src/universal_transfer_operator/data_providers/base.py

```diff
@@ -61,7 +65,7 @@ def write(self, source_ref):
         """Write the data from local reference location to the dataset"""
         raise NotImplementedError
 
-    def load_data_from_source_natively(self, source_dataset: Dataset, destination_dataset: Dataset) -> None:
+    def load_data_from_source_natively(self, source_dataset: T, destination_dataset: T) -> Any:
```

Comment:
What type of output would this method usually return?

Reply:
WDYT if we just removed it, while we don't need it - and then we can look into this again when we have native support?